web/client/utils/styleparser/__tests__/CesiumStyleParser-test.js

```diff
@@ -592,59 +592,6 @@ describe('CesiumStyleParser', () => {
             }).catch(done);
         });
 
-        it('should add leader line where HeightReference is Relative and sampleTerrain is using when from Cesium', (done) => {
```

Comment:
This test is not needed anymore?

Reply:
No, it is not needed. They removed the `Cesium.when` library so this case does not exist anymore